docs/PacketFence_PKI_Quick_Install_Guide.asciidoc

```diff
@@ -28,7 +28,7 @@ Installation
 Step 1: Install the PKI
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-NOTE: The PacketFence PKI application is available on the PacketFence repository as the package "packetfence-pki". This is available for CentOS6, Debian7 and Ubuntu 12.04.
+NOTE: The PacketFence PKI application is available on the PacketFence repository as the package "packetfence-pki". This is available for CentOS 6, Debian 7 and Ubuntu 12.04.
```

Comment:
What standard would you prefer to use.
* CentOSX
* CentOS-X
* CentOS X
My prefered one is 'CentOS X' however it really does not matter as it is just minutia.